src/python/WMCore/MicroService/MSPileup/MSPileupTasks.py

```diff
@@ -62,6 +62,31 @@ def pileupSizeTask(self):
             msg = f"MSPileup pileup size task failed with error {exp}"
             self.logger.exception(msg)
 
+    def cleanupTask(self, timeThreshold=15):
```

Comment:
I think this argument (timeThreshold) should be mandatory.

Reply:
done